### PR TITLE
Uds 1667 - Static Site Generator update as React App

### PR DESCRIPTION
**Pull request by scott-williams-az**
### Links

- [JIRA ticket](https://asudev.jira.com/browse/UDS-1667)
- [Unity reference site](https://asu.github.io/asu-unity-stack/)

### CODE REVIEW

**Comment by davidornelas11** [comment]
Screenshot 2024-10-29 at 5 00 46 PM
Static site package looks good. There's a weird style clash with z-index but it's unrelated.

**Review by mlsamuelson — COMMENTED** [review]
Linting errors.

Screenshot 2024-10-31 at 3 36 13 PM

File: packages/static-site/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# React + TypeScript + Vite
+
+- This app was created using the following code: `npm create vite@latest my-app -- --template react`
+
+- Similar to other quick start templates:
+`create-react-app`
+
+- Development server uses a custom host, which needs to be added to your machines host file `127.0.0.1 localhost.unity.web.asu.edu`
```
Comment (mlsamuelson) [inline packages/static-site/README.md:8]: Should be `127.0.0.1 localhost.asu.edu` to match what code expects and instructions on line 12.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
Header scroll behavior missing (shrinking on scroll). Not a deal-breaker for initial rollout, but maybe an easy fix?

Z-index, as noted by @davidornelas11 scolling over the header.

As noted, in conversation, to be solved: linking directly to routes, or refreshing the page after having browsed to a route should not return a broken/missing page.

**Comment by mlsamuelson** [comment]
Also, when I built and ran locally, the Storybook references weren't in place. Is that expected in this case, or indicative of work to be done?

**Comment by scott-williams-az** [comment]
The footer image path is defaulting to root. Haven't figure this out yet, I am hoping it is a simple fix in vite.config
image

**Comment by scott-williams-az** [comment]
replaced PR with https://github.com/ASU/asu-unity-stack/pull/1401

**Pull request closed without merging**